File: Facebook.NET/Contracts/GraphAPI.Error.cs
using System;
using System.Runtime.Serialization;

namespace Facebook
{
    public static partial class GraphApi
    {
        [DataContract]
        public sealed class Error
        {
            [DataMember(Name = "message")]
            public string Message { get; set; }

            [DataMember(Name = "type")]
            public string Type { get; set; }

            [DataMember(Name = "code")]
            public string Code { get; set; }

            [DataMember(Name = "error_subcode")]
            public string ErrorSubcode { get; set; }
        }
    }
}

File: Facebook.NET - VB/Contracts/GraphAPI.Error.vb
Imports System
Imports System.Runtime.Serialization

Namespace Facebook
    Public NotInheritable Partial Class GraphApi
        <DataContract()>
        Public NotInheritable Class [Error]
            <DataMember(Name:="message")>
            Public Property Message() As String
            <DataMember(Name:="type")>
            Public Property Type() As String
            <DataMember(Name:="code")>
            Public Property Code() As String
            <DataMember(Name:="error_subcode")>
            Public Property ErrorSubcode() As String
        End Class
    End Class
End Namespace